scripts/check_frontend_coverage.py

```diff
@@ -337,6 +337,7 @@
     'promo-bar.directive.ts',
     'promo-bar.service.ts',
     'python-program.tokenizer.ts',
+    'question-creation.service.ts',
```

Comment:
Why is this added? As mentioned before, you need to get @marianazangrossi's explicit approval for this.

Reply:
Hey @marianazangrossi @seanlip I spent almost 6-7 hours but was unable to get it to 100%, filed an issue and assigned that to me. Hope that's fine @marianazangrossi .

Reply:
I think this is ok, as an issue has been filed. However, please don't delay this by more than a week!